lib/src/dbs/capabilities.rs

```diff
@@ -192,7 +193,8 @@ impl<T: Target + Hash + Eq + PartialEq + std::fmt::Display> std::fmt::Display fo
 /// ```
 ///
 /// Create a new instance, and allow certain functions
-/// ```no_run
+#[cfg_attr(feature = "kv-rocksdb", doc = "```no_run")]
+#[cfg_attr(not(feature = "kv-rocksdb"), doc = "```ignore")]
```

Comment:
Same here. If we remove the RocksDB dependency these docs can be compiled and tested.